### PR TITLE
Update tiers

### PR DESCRIPTION
**Pull request by akarve**
# Description

# TODO

- [n/a] Unit tests
- [n/a] Automated tests (e.g. Preflight)
- [n/a] Documentation
    - [n/a] [Python: Run `build.py`](../gendocs/build.py) for new docstrings
    - [n/a] JavaScript: basic explanation and screenshot of new features
- [n/a] [Changelog](CHANGELOG.md) entry (skip if change is not significant to end users, e.g. docs only)

### CODE REVIEW

File: catalog/README.md [hunk 1]
```diff
@@ -35,3 +35,73 @@ but that's handled in a separate domain (`state.route`) via reducer composition;
...
+    "rules": "database.rules.json"
+  },
+  "hosting": {
+    "public": "build",
+    // route all paths to single page app
+    "rewrites": [{
+      "source": "**",
+      "destination": "/index.html"
+    }],
+		"ignore": [
```
Comment (fiskus) [inline catalog/README.md:58]: It seems like tabulation character here

**Review by fiskus — APPROVED** [review]

File: catalog/README.md [hunk 2]
```diff
@@ -35,3 +35,73 @@ but that's handled in a separate domain (`state.route`) via reducer composition;
...
 - An accurate check for a successful fetch() would include checking that the promise resolved, then checking that the Response.ok property has a value of true. The code would look something like this:
 [msdn fetch doc](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)
+
+## Deployment (for Quilt internal usage)
+```sh
+firebase use ENV
+firebase deploy --only hosting
+```
+
+### firebase.json
```
Comment (fiskus) [inline catalog/README.md:45]: I'd prefer to have a file `internals/firebase.default.json` or `internals/firebase.placeholder.json`

Comment (nl0) [reply catalog/README.md:45]: we can just put it into catalog root if that's where it's used from

Comment (akarve) [reply catalog/README.md]: I put it in internals/firebase.default.json for now.

Comment (akarve) [reply catalog/README.md]: It is used from /catalog, btw. Lmk.

**Review by nl0 — APPROVED** [review]

**Review by nl0 — APPROVED** [review]

**Pull request merged**